### PR TITLE
Added doc for alert, avatar, badge, button and card

### PR DESCRIPTION
**Pull request by muskanbagrecha**

### CODE REVIEW

File: alert.html [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    <button class="btn-floating float-on-screen bottom-right doc-float">
+        <i class="fas fa-arrow-up"></i>
+    </button>
+
+    <script src="https://kit.fontawesome.com/be9a255fe4.js" crossorigin="anonymous"></script>
+    <script src="js/prism.js"></script>
+    <script src="js/app.js"></script>
+</body>
+
+</html>
```
Comment (ShrutiLakhani) [inline alert.html:287]: Code looks clean and properly documented

File: badge.html [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+    <button class="btn-floating float-on-screen bottom-right doc-float">
+        <i class="fas fa-arrow-up"></i>
+    </button>
+
+    <script src="https://kit.fontawesome.com/be9a255fe4.js" crossorigin="anonymous"></script>
+    <script src="js/prism.js"></script>
+    <script src="js/app.js"></script>
+</body>
+
+</html>
```
Comment (ShrutiLakhani) [inline badge.html:352]: Clean. Well documented code

**Review by ShrutiLakhani — APPROVED** [review]
Code looks clean to merge

File: badge.html [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+
+                            </script>
+                        </code>
+                    </pre>
+                </div>
+
+            </div>
+
+            <!-- Edit badges -->
+            <div class="component__variant">
```
Comment (PuneetP16) [inline badge.html:166]: I didn't understand this comment can you please explain?

**Review by PuneetP16 — APPROVED** [review]
Well structured document, easy to understand where different components are, due to good comments.

File: button.html [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+                        <li>Note that buttons can float against the parent container or the viewport. For static
+                            position, use class <span class="highlight">btn-floating.</span></li>
+                        <li>For floating with respect to the parent container, add one more class which specifies
+                            the position: <span class="highlight">top-left, top-right, bottom-left,
+                                bottom-right.</span></li>
+                        <li>For floating button with respect to viewport, add class <span
+                                class="highlight">float-on-screen.</span> For example, the button on bottom right of
+                            the screen.</li>
+                        <li>Position can be customized by overriding the positions specified in the respective
+                            class.</li>
```
Comment (rahulyadav139) [inline button.html:246]: Docsite is well documented. Specified additional classes to the users which are highlighted also. Layout wise website is looking great.  Color scheme is awesome. Great work

**Review by rahulyadav139 — APPROVED** [review]

File: js/app.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    else{
+        themeIcon.classList.remove('fa-sun');
+        themeIcon.classList.add('fa-moon');
+    }
+}
+
+// for (let i = 0; i < themeIcons.length; i++) {
+//     themeIcons[i].addEventListener("click", switchStyle);  
+// }
+
```
Comment (PuneetP16) [inline js/app.js:35]: There's commented code, could be removed if not useful

File: js/app.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+let themeToggle = document.querySelector(".theme-toggle");
+let themeIcon = document.querySelector('.theme-toggle i');
+
+if (localStorage.getItem("lastActiveSheet")) {
+    activeSheet.setAttribute("href", localStorage.getItem("lastActiveSheet"));
+}
+
+function switchStyle() {
+    let selectedSheet = this.getAttribute("data-stylesheet");
+    console.log(selectedSheet);
```
Comment (PuneetP16) [inline js/app.js:13]: Suggestion: function is large, could add comments explaining it in brief or javascript docstring. 

Demo:
---
image

Here's some references to read more on that if you need,
https://jsdoc.app/about-getting-started.html
https://dev.to/stephencweiss/write-your-own-javascript-contracts-and-docstrings-42ho

File: js/app.js [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    else{
+        themeIcon.classList.remove('fa-sun');
+        themeIcon.classList.add('fa-moon');
+    }
+}
+
+// for (let i = 0; i < themeIcons.length; i++) {
+//     themeIcons[i].addEventListener("click", switchStyle);  
+// }
```
Comment (PuneetP16) [inline js/app.js:34]: here too.

**Review by PuneetP16 — APPROVED** [review]
All the sections are easy to navigate to, due to proper mentioning of comments, good work, Overall theme of all components looks consistent and pleasing to eyes. Looks good to merge.

**Pull request merged**